### PR TITLE
Add CMake install script

### PR DESCRIPTION
**Pull request by RamadanAhmed**

### CODE REVIEW

File: Source/C++/Core/Ap4Atom.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@
 #include "Ap4AtomFactory.h"
 #include "Ap4Debug.h"
 #include "Ap4UuidAtom.h"
-
+#include <cstdint>
```
Comment (barbibulle) [inline Source/C++/Core/Ap4Atom.cpp:39]: What type specifically what missing?

File: bento4Config.cmake [hunk 2]
```diff
@@ -0,0 +1,2 @@
+include(CMakeFindDependencyMacro)
```
Comment (barbibulle) [inline bento4Config.cmake:1]: This file seems to be by itself and not referenced anywhere

File: Source/C++/Core/Ap4Atom.cpp (hunk 1 above)
Comment (RamadanAhmed) [reply Source/C++/Core/Ap4Atom.cpp:39]: It was an issue in a previous commit, but it is solved now
I will remove it now

File: bento4Config.cmake (hunk 2 above)
Comment (RamadanAhmed) [reply bento4Config.cmake:1]: It is helpful in case of 3rd party dependencies but indeed it is not needed here

**Comment by hjmallon** [comment]
This PR https://github.com/axiomatic-systems/Bento4/pull/380/files is already a more complete version of the CMake install.

**Comment by barbibulle** [comment]
I'm closing this as it should no longer be needed since #621 has been merged. Let me know if that's not right.

**Pull request closed without merging**